### PR TITLE
Workflow for closing old prs added

### LINKED ISSUE #646: [Feature] <add GitHub Action to organize PRs more>
### Description

Hey @rupali-codes! :) I just found [this action feature on GitHub](https://github.com/marketplace/actions/close-stale-issues). I noticed that we get a lot of old PRs, so I think adding this action would organize things a bit. What do you think?

### Screenshots

_No response_

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [ ] I am willing to work on this issue (optional)
- [ ] I am a GSSoC'23 contributor

### PR DESCRIPTION
**Pull request by Tushar98644**
## Changes proposed

I have created a new github actions for closing old prs with a time limit of 30 days(can be changed as per convenience)

### CODE REVIEW

**Comment by Tushar98644** [comment]
@CBID2 @rupali-codes kindly review...

**Review by CBID2 — APPROVED** [review]
It looks pretty good @Tushar98644! 😊

**Comment by Tushar98644** [comment]
> It looks pretty good @Tushar98644! 😊

Thanks for reviewing so quickly😁 ....i haven't been assigned this issue yet btw😅

**Comment by CBID2** [comment]
Now you are @Tushar98644! 😊

**Review by rupali-codes — COMMENTED** [review]
awesome, can we make it 45 days?

**Comment by Tushar98644** [comment]
> awesome, can we make it 45 days? Also, can update contributin.md in the PR section, that the PR will get closed after 45days :)



> awesome, can we make it 45 days? Also, can update contributin.md in the PR section, that the PR will get closed after 45days :)

updated , now the old prs will close after 45 days of opening.....should I create a new issue for the updating the contributing.md as this will keep the prs clean.....

**Comment by rupali-codes** [comment]
> > awesome, can we make it 45 days? Also, can update contributin.md in the PR section, that the PR will get closed after 45days :)
> 
> > awesome, can we make it 45 days? Also, can update contributin.md in the PR section, that the PR will get closed after 45days :)
> 
> updated , now the old prs will close after 45 days of opening.....should I create a new issue for the updating the contributing.md as this will keep the prs clean.....

sure

File: .github/ISSUE_TEMPLATE/add_link.yml [hunk 1]
```diff
@@ -18,6 +18,7 @@ body:
     attributes:
       label: Category
       options:
+        - Open Source
```
Comment (rupali-codes) [inline .github/ISSUE_TEMPLATE/add_link.yml:21]:
```suggestion
```

**Review by rupali-codes — APPROVED** [review]
looks perfect

**Pull request merged**